docs/developers/dev-guide.md

```diff
@@ -163,7 +163,7 @@ withdrawn by the governance under the tutelage of Aragon court.
 ## Adding New Tokens
 Hermez contains a list with all tokens supported. The following list includes some requirements on the token listing:
 - Tokens must be ERC20
-- Only the governance can add new tokens.
+- Governance can decide the fee cost of adding tokens, therefore can regulate the token listing.
```

Comment:
remove the comma and replace with "and" - fee cost of adding tokens and therefore can regulate the token listing. 
:)

Reply:
ty!!